rust-runtime/aws-smithy-cbor/Cargo.toml

```diff
@@ -1,21 +1,32 @@
 [package]
 name = "aws-smithy-cbor"
 version = "0.0.0-smithy-rs-head"
-authors = ["AWS Rust SDK Team <aws-sdk-rust@amazon.com>", "David Pérez <d@vidp.dev>"]
+authors = [
+    "AWS Rust SDK Team <aws-sdk-rust@amazon.com>",
+    "David Pérez <d@vidp.dev>",
+]
 description = "CBOR utilities for smithy-rs."
 edition = "2021"
 license = "Apache-2.0"
 repository = "https://github.com/awslabs/smithy-rs"
 
 [dependencies.minicbor]
 version = "0.19.1" # TODO Update
-features = ["alloc"] # To write to a `Vec<u8>`: https://docs.rs/minicbor/latest/minicbor/encode/write/trait.Write.html#impl-Write-for-Vec%3Cu8%3E
+features = [
+    "alloc",
+    "half"  # To support reading `f16` to accomodate fewer bytes transmitted that fit the value.
+] # To write to a `Vec<u8>`: https://docs.rs/minicbor/latest/minicbor/encode/write/trait.Write.html#impl-Write-for-Vec%3Cu8%3E
 
 [dependencies]
 aws-smithy-types = { path = "../aws-smithy-types" }
+criterion = "0.5.1"
```

Comment:
Shouldn't this be a `dev-dependency`? https://bheisler.github.io/criterion.rs/book/getting_started.html#step-1---add-dependency-to-cargotoml